LeadDuplicator.js

```diff
@@ -1,17 +1,15 @@
-let duplicatesCount = 5;
-let formData = ZDK.Page.getForm().getValues(); // collecting the data from form
+const duplicatesCount = 5;
+const formData = ZDK.Page.getForm().getValues(); // collecting the data from form
+const leadObjects = [];
 
```

Comment:
Please add the following line to define leadName,
const leadName = formData["First_Name"];

Reply:
Resolved